tests/accounts_test.go

```diff
@@ -1,8 +1,10 @@
-package main
+package tests
```

Comment:
Tests should be implemented within the same package as the function they're testing. So within the controllers package, you should have a `*_test.go` file for every file which tests the functions inside that file. Additionally, the name of the package for these `*_test.go` files should be `package *_test`. In your case, because you're testing the controllers package, the name should be `package controllers_test`.